templates/taxi/car_list.html

```diff
@@ -1,14 +1,16 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list<a href = {% url "taxi:car-create" %}>+</a></h1>
   {% if car_list %}
     <ul>
       {% for car in car_list %}
         <li>
           <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
```

Comment:
There is an extra space before the closing quotation mark in the URL tag for the car detail link. It should be `{% url 'taxi:car-detail' pk=car.id %}` without the space before the closing quote.